src/main/java/com/messenger/exception/NotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.messenger.exception;
+
+
+import lombok.experimental.FieldDefaults;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@FieldDefaults(level = PRIVATE)
+public class NotFoundException extends RuntimeException {
```

Comment:
Spring data has a default EntityNotFoundException which is handled by Spring to return HTTP status 404